openwisp_radius/tests/test_saml/utils.py

```diff
@@ -0,0 +1,30 @@
+import base64
+
+from django.conf import settings
+from django.utils.module_loading import import_string
+from djangosaml2.cache import OutstandingQueriesCache
+
+
+class TestSamlUtilities:
```

Comment:
for consistency with the rest of the openwisp codebase we should call this something like `TestSamlMixins` or similar (see other test mixins used around the codebase)